eve/config.py

```diff
@@ -1,12 +1,19 @@
+import os
+
 # Weather API
-WX_API_KEY = "<YOUR-API-KEY>"
-WX_LOCATION = "<YOUR-DEFAULT-LOCATION>"
-WX_METRIC_TEMP = "<celsius|fahrenheit|kelvin>"
-WX_METRIC_WIND = "<miles_hour|km_hour|knots>"
+WX_API_KEY = os.environ.get("WX_API_KEY")
+WX_LOCATION = os.environ.get("WX_LOCATION", "London")
+WX_METRIC_TEMP = os.environ.get("WX_METRIC_TEMP", "celsius")
+WX_METRIC_WIND = os.environ.get("WX_METRIC_WIND", "km_hour")
 
 # Github API
-GITHUB_USER = "<USERNAME>"
-GITHUB_PASS = "<PASSWORD>"
+GITHUB_USER = os.environ.get("GITHUB_USER")
+GITHUB_PASS = os.environ.get("GITHUB_PASS")
```

Comment:
tbf - you've already supplied defaults for most...its is just the GitHub variables 😄